packages/jest-config/src/validConfig.js

```diff
@@ -55,9 +55,13 @@ module.exports = ({
   modulePathIgnorePatterns: ['<rootDir>/build/'],
   modulePaths: ['/shared/vendor/modules'],
   name: 'string',
+  noDefaultReporters: false,
```

Comment:
@cpojer I like the style @dmitriiabramov included for adding the default reporters. I can remove `.noDefaultReporters` and implement the reporters option first and additionalReporters pretty much going to be based on that so afterwards, even implementing additionalReporters won't be a big problem.